test/DynamoCoreWpfTests/NodeViewCustomizationTests.cs

```diff
@@ -314,9 +314,11 @@ public void WatchImageCoreContainsImage()
 
             // Starting from Dynamo 2.13, node view now comes with 
             // images like node icon, lacing image etc
-            Assert.AreEqual(3, imgs.Count());
+            // As of March 2022, we have 7 images per NodeView
+            // Images are named for ease of use
+            Assert.AreEqual(7, imgs.Count());
```

Comment:
any noticeable increase in memory use?

Reply:
I did a high level comparison watching the process memory in use with and without this change. My use case was to display 792 nodes. Without the change the average was ~1.23 Gb over three runs. ~1.24 Gb over three runs with the change. Looks OK to me.

Reply:
Thank you, Jorgen!